lua/nvim-tree/legacy.lua

```diff
@@ -268,12 +268,6 @@ local g_migrations = {
       o.respect_buf_cwd = vim.g.nvim_tree_respect_buf_cwd == 1
     end
   end,
-
-  nvim_tree_create_in_closed_folder = function(o)
```

Comment:
I think it's time to remove the `g:` legacy. Maybe after merging this...

Reply:
I am fine with that - it's been long enough.

Reply:
Fantastic!